velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -22,11 +22,73 @@
 
 namespace facebook::velox::substrait {
 
+class SubstraitToVeloxFilter {
+ public:
+  virtual void addFilter(
+      const std::string& filterName,
+      std::unique_ptr<common::DoubleRange>&& range) = 0;
```

Comment:
Why are we hard-coding DoubleRange here? There are other types of filters that need to be supported as well. Should we use common::Filter instead?

Reply:
Good point. I will look into this.